client/my-sites/ads/main.jsx

```diff
@@ -247,8 +251,19 @@ class AdsMain extends Component {
 			component = this.renderInstantActivationToggle( component );
 		}
 
+		const adsProgramName = isJetpack ? 'Ads' : 'WordAds';
+		const layoutTitles = {
+			earnings: translate( '%(wordads)s Earnings', { args: { wordads: adsProgramName } } ),
+			settings: translate( '%(wordads)s Settings', { args: { wordads: adsProgramName } } ),
```

Comment:
This existed before but do we not translate 'Ads' or 'WordAds'? cc @Automattic/i18n